modules/apps/headless/headless-builder/headless-builder-impl/src/main/resources/com/liferay/headless/builder/internal/batch/headless-builder.batch-engine-data.json

```diff
@@ -371,6 +371,65 @@
 			"modifiable": true,
 			"name": "APIProperty",
 			"objectFields": [
+				{
+					"DBType": "String",
+					"businessType": "Picklist",
+					"defaultValue": "normal",
+					"externalReferenceCode": "API_PROPERTY_TYPE",
+					"indexed": true,
+					"indexedAsKeyword": false,
+					"label": {
+						"en_US": "API Property Type"
+					},
+					"listTypeDefinitionExternalReferenceCode": "API_PROPERTY_TYPE_PICKLIST",
+					"name": "apiPropertyType",
+					"objectFieldSettings": [
```

Comment:
I think that we should have to include it, because if we don't indicate the `stateFlow` property, no value will be created as ListEntry. As well as, it will throw an Exception that we have indicated the defaultValue but there are no values indicated. I don't know if you are testing it with PickList that have been already created and link it with an Object Defintion, but in this case we are starting from scratch and we have to define the different types of options.